datahub_emails/api.py

```diff
@@ -26,3 +28,71 @@ def on_new_user(user_info):
         logging.error(errors[0])
         return False
     return True
+
+
+def on_incident(incident, publisher, errors='', status='identified'):
+    if isinstance(errors, list):
+        errors = '\n'.join(errors)
+    component_id = _get_component_id(publisher)
+    logging.error(component_id)
+    if component_id is None:
+        logging.info('Component with name %s Not Found' % publisher)
+        return
+
+    payload = {
+      "incident": {
+        "name": incident,
+        "body": errors,
+        "status": status,
+        "component_ids": [component_id]
+      }
+    }
+    resp = requests.post(
+        'https://api.statuspage.io/v1/pages/{page_id}/incidents'.format(
+                                                    page_id=statuspage_id),
+        headers=statuspage_headers,
+        data=json.dumps(payload)
+    )
+    if resp.status_code > 205:
+        logging.info('Not able to connect to https://api.statuspage.io')
+        return
+    return resp.json()
+
+
+def subscribe_user(username, email):
+    component_id = _get_component_id(username)
+    if component_id is None:
+        logging.info('Component with name %s Not Found' % username)
+        return
+
+    payload = json.dumps({
+      "email": email,
+      "component_ids": [
+        component_id
+      ]
+    })
+    resp = requests.post(
+        'https://api.statuspage.io/v1/pages/{page_id}/subscribers'.format(page_id=statuspage_id),
+        headers=statuspage_headers,
+        data=payload
```

Comment:
I think requests has a `json` parameter exactly to avoid the need to use `json.dumps`